### ISSUE TITLE
bug: `checkly test --list` runs into type error

### ISSUE THREAD

**Issue by tdeekens** [checkly/checkly-cli#677]
### Node.js version

19

### NPM version

9

### @checkly/cli version

4.0.3

### Steps to reproduce

1. Setup some test
2. Run `checkly test --list` inside the project

### What is expected?

The checks setup in the project are listed

### What is actually happening?

An error is thrown

```bash
Parsing your project... done

Listing all checks:

    TypeError: Cannot read properties of undefined (reading 'size')
```

### Any additional comments?

This happens only with the recently released version of the CLI of v4.

**Comment by tnolet** [comment]
@tdeekens thanks for reporting. That slipped through. We will fix it and add a test!

**Comment by tdeekens** [comment]
No worries. Thanks for reacting to quickly.

**Comment by clample** [comment]
@tdeekens thank you again for reporting this! We made a new release which fixes this (version `4.0.4`).
